_pkgdown.yml

```diff
@@ -7,3 +7,337 @@ template:
   bootstrap: 5
   params:
     bootswatch: flatly
+  opengraph:
+    twitter:
+      creator: "@alexpghayes"
+      card: summary
+
+reference:
+  - title: "Generic functions for S3 distribution objects"
+    contents: 
+      - pdf
+      - cdf
+      - random
+      - variance
+      - skewness
+      - kurtosis
+      - support
+      - is_continuous
+      - is_discrete
+      - suff_stat
+      - fit_mle
+      - is_distribution
+      - prodist
+  - title: "Utilities, tools, and data"
+    contents: 
+      - apply_dpqr
+      - make_support
+      - make_positive_integer
+      - likelihood
+      - log_likelihood
+      - FIFA2018
+  - title: "Visualization infrastructure"
+    contents: 
+      - plot.distribution
+      - plot_cdf
+      - plot_pdf
+      - geom_auc
+  - title: "Bernoulli distribution"
+    contents:
+      - Bernoulli
```

Comment:
If you don't want to write these all out by hand you can just do `matches("Bernoulli")`. Probably a bit easier. https://pkgdown.r-lib.org/reference/build_reference.html#topic-matching

Reply:
Caveats: (a) we would need to be a bit more careful than just using `matches()` or `ends_with()` (`Poisson` vs. `HurdlePoisson` vs. `ZIPoisson` etc.), (b) the distribution generator function should always be listed first in each section.

In our case, (b) might be a non-issue in our case because the upper-case generator functions seem to be sorted before the lower-case methods. Not sure, though, whether this is the case on all platforms/locales.

If you think it's worth the effort, I can look into (a). I would probably leave it as it is.

Reply:
Ok, that's fine by me!

Reply:
Great, thanks! We can still revisit this at a later point if we feel that maintaining the `_pkgdown.yml` is too much of a hassle.